microsoft-edge/toc.yml

```
@@ -351,6 +351,12 @@
               href: extensions-chromium/publish/manage-settings.md
             - name: Contact extensions support
               href: extensions-chromium/publish/contact-extensions-team.md
+            - name: Microsoft Edge Add-ons Publish API
+              items:
+                - name: Using the Add-ons Publish API
```

```suggestion
                - name: Using the Publish Add-ons API
```
ok for a short version of the article title?